subsys/nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -2436,7 +2436,9 @@ psa_status_t psa_driver_wrapper_get_random(psa_driver_random_context_t *context,
 					   size_t output_size)
 {
 #if defined(PSA_CRYPTO_DRIVER_ALG_PRNG_TEST)
-	return prng_test_generate_random(output, output_size);
+	if (prng_test_generate_random(output, output_size) == PSA_SUCCESS) {
+		return PSA_SUCCESS;
+	}
```

Comment:
Let's be consistent with error handling in this function.

So,

int err;

err = bla_bla();

if(err == PSA_SUCCESS) {
return SUCCESS;
}

Reply:
Sure, I will update the code